### ISSUE TITLE
Nest <va-promo-banner> within <header>

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15943]
## Description
The `<va-promo-banner>` should be nested under the `<header>` This currently isn't being flagged as an error because it has it's own landmark role being applied to it. However, this shouldn't be there and [I've asked that it be removed](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2239). I would like to move this under the `<header>` during the redesign because that's where it should be to ensure all content is contained by a landmark.

## Steps for Implementation
Move the `<va-promo-banner>` directly underneath the `<div id="legacy-header">`:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/2fdb5d8f-15a4-4018-aaaa-3f47a122f02c)

I think this is ok for the injected header too because just the Legacy Header gets injected from what I was seeing but obviously this needs to be confirmed

## Acceptance Criteria
- [ ] The `<va-promo-banner>` is wrapped within the `<header>`
- [ ] The injected header is not impacted
- [ ] Requires design review
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [x] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by jilladams** [comment]
Noting: from Laura's perspective, this isn't hugely critical, just nice to fix early if we can. If we determine that the testing burden is high and should wait til we implement the Federal header / footer, that's ok.

**Comment by chriskim2311** [comment]
@laflannery Can you take a quick look at this before I close. I was able to see the correct html markup for the va-promo-banner.

**Comment by laflannery** [comment]
Looks good @chriskim2311 - It is correctly under the Header (aka banner landmark) and no additional axe errors because the DST update shipped too, gold stars for everyone!!

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/506af98f-adcc-40e5-9cb1-b03e7e8ac279)
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/a159cb84-7544-449f-aae3-21709281d4ef)

**Comment by chriskim2311** [comment]
Great thanks for taking a look closing!
